tests/fs/fallback/s3/s3_fallback_fixtures.py

```diff
@@ -10,7 +10,7 @@ def fs(self):
         return AlluxioFileSystem(
             etcd_hosts="localhost",
             target_protocol="s3",
-            test_options={"skip_alluxio": True},
```

Comment:
@SibylYang the reason we adding test_options is to put some shaded values here. please do not modify the user behavior for now because some of our users are relying on the existing configurations.
Please do not modify the end-user behavior